CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # CHANGELOG
 
-All notable changes to this project will be documented in this file.
+All notable changes to this project were documented in this file. This is of historical interest only at this point, and changes are now logged per gem (as of the 0.6.0 release).
```

Comment:
Is this statement OK, given that the new tooling now updates gem-specific changelogs?

Reply:
I agree, I think this makes sense.